src/base/runners.ml

```diff
@@ -215,7 +217,7 @@ struct
           Runner.State.make ~num_inputs ~input ~next_auxiliary ~aux ~system
             ~with_witness:false ()
         in
-        let id = Run_state.id state in
+        let id = Backend.Run_state.id state in
```

Comment:
Nit: this doesn't seem to be useful (compiles without on my side), as the module is opened L21. Or, are you trying to remove the statements `open` :face_holding_back_tears:?